app/cdap/components/hydrator/reducers/DAGNodesStore.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { createAction, createReducer, current } from '@reduxjs/toolkit';
+import { GLOBALS } from 'services/global-constants';
+import uuidV4 from 'uuid/v4';
+import { santizeStringForHTMLID } from 'services/helpers';
+
+const addNode = createAction<any>('ADD_NODE')
+const updateNodeUIPosition = createAction<any>('UPDATE_NODE_UI_POSITION')
+const reset = createAction<any>('LEFTPANELSTORE_RESET');
```

Comment:
Is there anything we can do to add a more specific type than `any`? Even a simple base type?